content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -98,21 +98,6 @@ For relational databases, the available options are:
 
 For MongoDB, the only available option is the `prisma` relation mode. This mode is also active if no `relationMode` is explicitly set in the `datasource` block.
 
-Setting the relation mode is currently a preview feature. To enable this feature, add the `referentialIntegrity` feature flag to the `previewFeatures` list in the [`generator`](/reference/api-reference/prisma-schema-reference#generator) block:
-
-```prisma file=schema.prisma highlight=9;add
-datasource db {
-  provider     = "mysql"
-  url          = env("DATABASE_URL")
-  relationMode = "prisma"
-}
-
-generator js {
-  provider        = "prisma-client-js"
-  previewFeatures = ["referentialIntegrity"]
-}
-```
```

Comment:
This is the one that should probably replaced with a mention between which versions the preview feature flag was necessary for later reference.

Reply:
I've updated the admonition to have the version info